src/core/thread/router_table.cpp

```diff
@@ -261,6 +261,8 @@ otError RouterTable::Release(uint8_t aRouterId)
     ThreadNetif &netif  = GetNetif();
     uint16_t     rloc16 = Mle::Mle::GetRloc16(aRouterId);
 
+    VerifyOrExit(aRouterId <= Mle::kMaxRouterId, error = OT_ERROR_INVALID_ARGS);
```

Comment:
Returning `OT_ERROR_INVALID_ARGS` is not documented. Shall we verify this in `src/core/api/thread_ftd_api.cpp` and `assert()` it here?